.github/workflows/push-tests.yml

```diff
@@ -39,6 +39,7 @@ jobs:
       - name: Lint src with eslint
         working-directory: ./frontend
         run: |
+          npm run prettier-check
```

Comment:
Add this in PR GH Action to make sure `prettier` committed files have been checked by `prettier`, which is now done automatically at commit time via `lint-staged`